### PR TITLE
DO NOT MERGE - test PR for feedback

### PR DESCRIPTION
**Pull request by bshannon**
Is this what we expect a PR to look like?

## Specification PR template
Include the following:
- [x] Specification PDF in the form of wombat_x.y.z.pdf
- [x] Specification HTML in the form of wombat_x.y.z.html
- [x] Specification JavaDoc in the form of wombat-javadoc-x.y.z.zip
- [x] A root specification page following the template at: https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md
- [ ] For a Progress Review, that sufficient progress has been made on a Compatible Implementation and TCK, to ensure that the spec is implementable and testable.
- [ ] For a Release Review, a summary Compatible Implementation is complete, passes the TCK, and that the TCK includes sufficient coverage of the specification. The TCK users guide MUST include the instructions to run the compatible implementations used to validate the release. Instructions MAY be by reference.
- [x] The URL of the OSSRH staging repository for the api, javadoc. https://oss.sonatype.org/content/repositories/staging/jakarta/mail/jakarta.mail-api/1.6.4/

### CODE REVIEW

File: mail/1.6/mail.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+summary: "First release for Jakarta EE 8"
+---
+Jakarta Servlet defines server-side handling for HTTP requests and responses.
+
+* [Jakarta Mail 1.6 Specification Document](./mail-spec-1.6.pdf) (PDF)
+* [Jakarta Mail 1.6 Specification Document](./mail-spec-1.6.html) (HTML)
+* [Jakarta Mail 1.6 Javadoc](./apidocs)
+* [Jakarta Mail 1.6 TCK](http://download.eclipse.org/ee4j/mail-tck/1.6.0/release/mailtck-1.6.0_10-Jul-2019.zip)
+* Maven coordinates
+ * jakarta.mail:jakarta.mail-api:jar:1.6.4
```
Comment (ggam) [inline mail/1.6/mail.md]: What about adding a link to https://search.maven.org/artifact/jakarta.mail/jakarta.mail-api/1.6.4/jar?

Comment (bshannon) [reply mail/1.6/mail.md]: I suppose we could turn the Maven coordinates text into a link.
Should it go to the search results page, or directly to the download link for the jar file?

Comment (ggam) [reply mail/1.6/mail.md]: The search page contains ready to use code snippets for every major build system. On the other hand, that URL might change over time, although it's unlikely.

**Comment by bshannon** [comment]
Closing this test PR.

**Pull request closed without merging**